.circleci/config.yml

```diff
@@ -217,6 +217,19 @@ jobs:
             ./.circleci/build_image.sh gecko/Dockerfile gitpod/workspace-gecko
           no_output_timeout: 60m
 
+  workspace-nix:
```

Comment:
Please also register this new job in the list at the bottom of this file (in order to make CircleCI actually run it). 🙏